src/Diagnostics.DataProviders/DataProviders/AppInsightsDataProvider.cs

```diff
@@ -110,10 +114,10 @@ public async Task<DataTable> ExecuteAppInsightsQuery(string query, string operat
 
         public async Task<DataTable> ExecuteAppInsightsQuery(string query)
```

Comment:
If there is no purpose for this method then maybe just have the above `ExecuteAppInsightsQuery(string query, string operationName = null)`. Not sure if that'll break detectors..

Reply:
Changing the signature to  `ExecuteAppInsightsQuery(string query, string operationName = null)` breaks existing detectors hence we always added new functions rather than modifying the signature of the existing ones.